governance.md

```diff
@@ -326,15 +326,26 @@ Partners, with associated benefits:
 Changes to the governance documents are submitted via a GitHub pull
 request to The Project's governance documents GitHub repository at
 [https://github.com/jupyter/governance](https://github.com/jupyter/governance).
-The pull request is then refined in response to public comment and
-review, with the goal being consensus in the community.  After this
-open period, a Steering Council Member proposes to the Steering
-Council that the changes be ratified and the pull request merged
-(accepting the proposed changes) or proposes that the pull request be
-closed without merging (rejecting the proposed changes).  The Member
-should state the final commit hash in the pull request being proposed
-for acceptance or rejection and briefly summarize the pull request. All votes
-are limited in time to 4 weeks after the vote is initiated. At the end of 4
+There are two phases to the voting process:
```

Comment:
```suggestion
There are two phases to the process:
```
the voting phase is the second phase, so it's confusing to call the whole process a "voting" process.